project/NativeImage.scala

```diff
@@ -266,15 +268,16 @@ object NativeImage {
         sb.append(str + System.lineSeparator())
       })
       log.info(
-        s"Started building $artifactName native image. The output is captured."
+        s"Started building $targetLoc native image. The output is captured."
       )
-      val retCode = process.!(processLogger)
-      if (retCode != 0) {
-        log.error("Native Image build failed, with output: ")
+      val retCode    = process.!(processLogger)
+      val targetFile = artifactFile(targetDir, name, true)
+      if (retCode != 0 || !targetFile.exists()) {
+        log.error("Native Image build of $targetFile failed, with output: ")
```

Comment:
@JaroslavTulach missing `s` prefix

Reply:
Thanks E-Hern, I'll fix this as part of #10536:
```
commit 1bf17a5d562324c3f3736c0a3476e38511dca329 (HEAD -> wip/jtulach/NoLanguageServerDependency10010)
Author: Jaroslav Tulach <jaroslav.tulach@enso.org>
Date:   Fri Jul 12 07:28:04 2024 +0200

    Strings with $ need to be prefixed by s
```